features/step_definitions/back_office/registration_steps.rb

```diff
@@ -68,7 +68,7 @@
     expect(info_panel).to have_text("Carrier, broker and dealer")
   elsif @carrier == "broker_dealer"
     expect(info_panel).to have_text("Broker and dealer")
-  else
+  elsif @tier == "upper"
```

Comment:
Should this be `@carrier == "carrier_dealer"` instead? This `@tier == "upper"` feels a bit out of place in this `if` structure. I assume we don't set a `@carrier` if we deal with a `lower` tier as it doesn't apply there, hence 
```suggestion
  elsif @carrier == "carrier_dealer"
```
shall make the same result?